tests/LtiServiceConnectorTest.php

```diff
@@ -12,8 +14,10 @@ class LtiServiceConnectorTest extends TestCase
     public function testItInstantiates()
     {
         $registration = Mockery::mock(LtiRegistrationInterface::class);
+        $cache = Mockery::mock(Cache::class);
```

Comment:
Now that we're not relying on curl, can we add some additional automated tests here?

Reply:
Are these automated tests? they feel more like unit tests? Regardless, mocking the function calls may get a bit hairy. Might need to possibly pair.

Reply:
Unit testing this was a head scratcher. I added a test to make sure that a cached token could be retrieved. Just to get this moving, I think that's okay for now. We didn't previously have a test for these methods. Something to look into is how we can add resources to better test this library.